### PR TITLE
ios-cmake: add 4.1.1 and support new platforms

### PR DESCRIPTION
**Pull request by mologie**
Specify library name and version:  **ios-cmake/4.1.1**

Note: This also adds ios-cmake 4.1.0; version 4.1.1 was released while this PR was in review.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by dmn-star** [comment]
Maybe use the 4.1.1 version right away?
https://github.com/leetal/ios-cmake/releases/tag/4.1.1

**Comment by mologie** [comment]
Sure, I will update this PR later today. 4.1.1 was not yet on the GitHub releases page when I opened this PR (though the commit was there already, hence the "3 days ago" timestamp by GitHub).

File: recipes/ios-cmake/all/conandata.yml [hunk 1]
```diff
@@ -5,3 +5,9 @@ sources:
   "4.0.0":
     url: "https://github.com/leetal/ios-cmake/archive/4.0.0.zip"
     sha256: "2e0dcf00cd46ae808d85666ce4ce38ecd8875925588647295d41e97c874ad0b3"
+  "4.1.0":
+    url: "https://github.com/leetal/ios-cmake/archive/4.1.0.zip"
+    sha256: "13f32d364c2130469b1780c7b2d4914bab29f20230871202eb43c4f6665ccd29"
```
Comment (prince-chrismc) [inline recipes/ios-cmake/all/conandata.yml:10]:
```suggestion
```

File: recipes/ios-cmake/config.yml [hunk 2]
```diff
@@ -3,3 +3,7 @@ versions:
     folder: all
   "4.0.0":
     folder: all
+  "4.1.0":
+    folder: all
```
Comment (prince-chrismc) [inline recipes/ios-cmake/config.yml:7]:
```suggestion
```

**Review by prince-chrismc — CHANGES_REQUESTED** [review]
Unless explicitly required, we shy away from adding older versions

File: recipes/ios-cmake/all/conandata.yml (hunk 1 above)
Comment (mologie) [reply recipes/ios-cmake/all/conandata.yml]: Ack, applied your changed and rebased it to add 4.1.1 only.

**Review by prince-chrismc — APPROVED** [review]

File: recipes/ios-cmake/all/conandata.yml (hunk 1 above)
Comment (prince-chrismc) [reply recipes/ios-cmake/all/conandata.yml]: You can click the commit suggestion button  if you like its easier for me

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**